webpack.config.dev.js

```diff
@@ -22,8 +22,24 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['babel'],
-      include: path.join(__dirname, 'source')
+      include: path.join(__dirname, 'source'),
+      loader: 'babel',
+      query: {
```

Comment:
Better to use .babelrc for this.

If you want particular config to be applied only for dev env, then use `env` in`.babelrc`:

```json
{
  "env": {
    "development": {
       "plugins": []
     }
  }
}
```

https://babeljs.io/docs/usage/babelrc